### ISSUE TITLE
[Feature request] Please Colab with scene transition

### ISSUE THREAD

**Issue by Zarhog** [zeel01/poi-teleport#9]
The scence transition mod would be a great addiotion to this mod, allowign for deeper and more interesting sory telling.

**Comment by zeel01** [comment]
Could you elaborate a bit on what you would envision for this feature?

**Comment by Zarhog** [comment]
yes sorry about the vague desctiontion. the modal scene transition adds an extra button on scene controls which player attacked journal note as a slide both image and text before moving the map to the active scene. the idea would be to add the button as well to your annotation when clicking on the journal note on maps
